sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -527,27 +492,10 @@ export class EventHubReceiver extends LinkEntity {
     this._isStreaming = false;
 
     const cleanupBeforeAbort = (): Promise<void> => {
-      logger.info(
-        `[${this._context.connectionId}] The request operation on the Receiver "${this.name}" with address "${this.address}" has been cancelled by the user.`
-      );
+      logAbort(this._context.connectionId, this.name, this.address);
       return this.close();
     };
 
-    const tryOpenLink = (): Promise<void> =>
-      createAbortablePromise(
-        (resolve, reject) =>
-          this.initialize({
-            abortSignal,
-            timeoutInMs: getRetryAttemptTimeoutInMs(this.options.retryOptions),
-          })
-            .then(resolve)
-            .catch(reject),
-        {
-          abortSignal,
-          abortErrorMsg: StandardAbortMessage,
-        }
-      );
```

Comment:
`this.initialize` already takes an abort signal so no need to wrap it in a another abortable promise.